scripts/docker/README.md

```diff
@@ -0,0 +1,76 @@
+# Sandia-OpenSHMEM Docker Image
+Sandia-OpenSHMEM (SOS) utlizes Docker images to provide users a means to create
+containers that provide a configurable Ubuntu sandbox for running SOS
+applications and benchmarks.
+
+## Creating SOS Containers from the Prebuilt Docker Images
+Coming Soon: Information regarding the SOS public Docker Hub repos and prebuilt
+images.
+
+## Creating SOS Containers from the Dockerfile 
+This Dockerfile provides a configurable Ubuntu sandbox for running SOS
+applications and benchmarks.
+
+### Building the Image from the Dockerfile
+To build the image from the Dockerfile, run the following run the following
```

Comment:
I an seeing "run the following" twice here... I am having some eye issues recently and if you dont see the same on your end, please ignore.